test/TempleLineOfCredit.t.sol

```diff
@@ -0,0 +1,292 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import {ERC20Mock} from "openzeppelin-contracts/mocks/ERC20Mock.sol";
+import "../src/TempleLineOfCredit.sol";
+import "../src/common/access/Operators.sol";
+
+contract TempleLineOfCreditTest is Test {
+
+    TempleLineOfCredit public tlc;
+
+    uint256 public interestRateBps;
+    uint256 public minCollateralizationRatio;
+    uint256 public interestRatePeriod;
+
+    ERC20Mock public collateralToken;
+    uint256 public collateralPrice;
+
+    ERC20Mock public debtToken;
+    uint256 public debtPrice;
+
+    address admin = address(0x1);
```

Comment:
Can use `makeAddr("admin")` to add a label which is useful in debugging

https://book.getfoundry.sh/reference/forge-std/make-addr#makeaddr

bob and alice have the same address here...